SwSpotify/spotify_web.py

```diff
@@ -0,0 +1,75 @@
+import json
+import threading
+from http.server import HTTPServer, BaseHTTPRequestHandler
+
+"""
+Module to collect the current track data from the chrome extension if open.
+"""
+
+
+class Server(HTTPServer):
+
+    def __init__(self, server_address, RequestHandlerClass, bind_and_activate=True):
+        super().__init__(server_address, RequestHandlerClass, bind_and_activate=True)
+        self.data = {}
+
+    def finish_request(self, request, client_address):
+        self.data = self.RequestHandlerClass(request, client_address, self).data
+        pass
+
+
+class RequestHandler(BaseHTTPRequestHandler):
+
+    def __init__(self, *args):
+        super(RequestHandler, self).__init__(*args)
+        # self.data = None
+
+    def _set_headers(self):
+        self.send_response(200)
+        self.send_header("Access-Control-Allow-Headers", "*")
+        self.send_header("Access-Control-Allow-Origin", "*")
+        self.end_headers()
+
+    def do_OPTIONS(self):
+        self._set_headers()
+        self.data = {}
+
+    def do_POST(self):
+        # Handled posted data
+        content_length = int(self.headers['Content-Length'])
+        post_data = self.rfile.read(content_length)
+        self.data = json.loads(post_data)
+        # Send response
+        self._set_headers()
+        self.wfile.write(b'test')
+
+
+def server():
+    server_address = ('localhost', 5042)
```

Comment:
5042 is the port that `swaglyrics -t` runs on

Reply:
`5042` is the port number that the `SwagLyrics for spotify` extension was sending the data on. Should I change both the sending and receiving port in this fork and my fork of the extension? @aadibajpai

Reply:
Yes, it was 5042 because earlier SwSpotify wasn't an independent module by itself

Reply:
changed to port `5043`